zebra-chain/src/parameters/network/subsidy.rs

```diff
@@ -40,16 +40,27 @@ pub const BLOSSOM_POW_TARGET_SPACING_RATIO: u32 = 2;
 /// `(60 * 60 * 24 * 365 * 4) / 150 = 840960`
 pub const PRE_BLOSSOM_HALVING_INTERVAL: HeightDiff = 840_000;
 
+/// The halving height interval in the regtest is 6 hours.
+/// [zcashd regtest halving interval](https://github.com/zcash/zcash/blob/v5.10.0/src/consensus/params.h#L252)
+pub const PRE_BLOSSOM_REGTEST_HALVING_INTERVAL: HeightDiff = 144;
```

Comment:
Could we make these constants private to the module to ensure they can't be used directly? (I just made this mistake now in a suggestion PR).